clients/venice-admin-tool/src/main/java/com/linkedin/venice/AdminTool.java

```diff
@@ -1648,6 +1649,27 @@ private static void checkMigrationStatus(CommandLine cmd) {
     ControllerClient destControllerClient = new ControllerClient(destClusterName, veniceUrl, sslFactory);
 
     ChildAwareResponse response = srcControllerClient.listChildControllers(srcClusterName);
+
+    for (VeniceSystemStoreType systemStoreType: VeniceSystemStoreType.values()) {
+      MultiStoreInfoResponse srcClusterStoreInfo = srcControllerClient.getClusterStores(srcClusterName);
```

Comment:
I may have given bad advice, apologies :P 

It looks like the behavior of getClusterStores has an unfortunate behavior today.  In order for it to work, it has to be wired to the controller which happens to be the leader for administering that cluster.  This... is an annoying thing for an operator to figure out.  So need to restructure this a bit.

What we can is leverage another method in this class called queryStoreList.  This will consult the operator passed in controller endpoint for the complete store list.  We can then filter it down to the system store's we care about and print the migration status.

Sorry about that -_-;

Reply:
No worries!

I've made the modifications. 

Is there any way we can assert that `checkMigrationStatus()` is working properly in the test?

Also, how do I view the log output when running tests?

Thanks!